frontend/src/types.ts

```diff
@@ -1995,3 +1998,8 @@ export interface ExportedAssetType {
     has_content: boolean
     filename: string
 }
+
+export enum ConsoleFeedbackOptionValue {
```

Comment:
Nit: I wonder if this should be made more generic. Something like "YesOrNoResponse"